server/column.go

```diff
@@ -23,18 +25,17 @@ const maxColumnNameSize = 256
 
 // ColumnInfo contains information of a column
 type ColumnInfo struct {
-	Schema             string
-	Table              string
-	OrgTable           string
-	Name               string
-	OrgName            string
-	ColumnLength       uint32
-	Charset            uint16
-	Flag               uint16
-	Decimal            uint8
-	Type               uint8
-	DefaultValueLength uint64
-	DefaultValue       []byte
+	Schema       string
+	Table        string
+	OrgTable     string
+	Name         string
+	OrgName      string
+	ColumnLength uint32
+	Charset      uint16
+	Flag         uint16
+	Decimal      uint8
+	Type         uint8
+	DefaultValue interface{}
```

Comment:
```suggestion
	DefaultValue any
```

We can use new keyword ```any``` here.